### PR TITLE
Implemented RFC0010

### PR DESCRIPTION
**Pull request by pksunkara**
Please merge https://github.com/refractproject/rfcs/pull/32 when this is merged.

### CODE REVIEW

File: namespaces/api-description-namespace.md [hunk 1]
```diff
@@ -458,7 +472,23 @@ HTTP request message.
 - `element`: httpRequest (string, fixed)
 - `attributes`
     - `method` (string) - HTTP request method. The method value SHOULD be inherited from a parent transition if it is unset.
-    - `href` (Href) - A concrete URI for the request. The href SHOULD be inherited from a parent transition by expanding the href and hrefVariables if unset.
+    - `href` (Templated Href) - URI Template for this HTTP request.
+
+        If present, the value of the `href` attribute SHOULD be used to resolve
+        target URI of the http request.
```
Comment (kylef) [inline namespaces/api-description-namespace.md]: I think the following part doesn't flow very well:

> used to resolve target URI

Perhaps the following would sound better?

``` diff
- be used to resolve target URI of the http request.
+ be used to resolve the target URI of the HTTP request.
```

**Comment by pksunkara** [comment]
@kylef Updated.

**Comment by kylef** [comment]
:+1: Looks good to merge (I don't have access to merge it)

**Comment by smizell** [comment]
@pksunkara could you please bump the version on the README?

**Comment by pksunkara** [comment]
Done.

**Comment by kylef** [comment]
Good point @smizell, do we need to update the CHANGELOG too?

**Comment by smizell** [comment]
@kylef That would be good, though I'm afraid in my haste I did not update it.

**Comment by smizell** [comment]
@pksunkara Yeah, would you care to add 0.6.0 in the changelog and put what changed? I'll go back and fix the old stuff.

**Comment by pksunkara** [comment]
Updated. Also, please tag v0.6.0 when this PR is merged. Don't forget to merge the PR in RFCs repo.

**Pull request merged**